execute_rest_extractor/ice_cream_factory_datapoints_extractor/extractor.py

```diff
@@ -72,9 +72,9 @@ def run_extractor(
         stop_event: Cancellation token, will be set when an interrupt signal is sent to the extractor process
     """
 
-    config.frontfill.enabled = os.getenv("FRONTFILL_ENABLED", config.frontfill.enabled)
+    config.frontfill.enabled = os.getenv("FRONTFILL_ENABLED", config.frontfill.enabled).lower() == "true"
```

Comment:
@pavelzubarev Now I remember why I changed it. I used to get an error for boolean does not have .lower() method. But I'm don't know if it's actually being converted to bool correctly.

Reply:
made these explicit, so just in case there will be several type conversions, but in the end it should be predictable